File: src/main/java/no/nav/syfo/controller/domain/Bruker.java
package no.nav.syfo.controller.domain;


import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;
import no.nav.syfo.controller.domain.adresse.Bostedsadresse;
import no.nav.syfo.controller.domain.adresse.MidlertidigAdresseNorge;
import no.nav.syfo.controller.domain.adresse.MidlertidigAdresseUtland;
import no.nav.syfo.controller.domain.adresse.PostAdresse;
import java.io.Serializable;

@Data
@Accessors(fluent = true)
@EqualsAndHashCode
public class Bruker implements Serializable {
    public String navn;
    public Kontaktinfo kontaktinfo;
    public String arbeidssituasjon;
    public Bostedsadresse bostedsadresse;
    public MidlertidigAdresseNorge midlertidigAdresseNorge;
    public MidlertidigAdresseUtland midlertidigAdresseUtland;
    public PostAdresse postAdresse;
}

File: src/main/kotlin/no/nav/syfo/controller/domain/Bruker.kt
package no.nav.syfo.controller.domain

import no.nav.syfo.controller.domain.adresse.*
import java.io.Serializable

data class Bruker(
    val navn: String? = null,
    val kontaktinfo: Kontaktinfo? = null,
    val arbeidssituasjon: String? = null,
    val bostedsadresse: Bostedsadresse? = null,
    val midlertidigAdresseNorge: MidlertidigAdresseNorge? = null,
    val midlertidigAdresseUtland: MidlertidigAdresseUtland? = null,
    val postAdresse: PostAdresse? = null
) : Serializable
